### PR TITLE
Support-1.1 update dev tooling

### PR DESCRIPTION
**Pull request by seanfarmar**
This updates development tooling on the `support-1.1.1` branch.
Note: This should not be squashed.

### CODE REVIEW

**Comment by danielmarbach** [comment]
This branch has conflicts. @seanfarmar could update this PR?

**Comment by seanfarmar** [comment]
@danielmarbach it was against the wrong branch... fixed now

File: src/NServiceBus.Persistence.TestRunner/NServiceBus.Persistence.TestRunner.csproj [hunk 1]
```diff
@@ -14,8 +14,10 @@
     <PackageReference Include="Microsoft.ServiceFabric.FabricTransport.Internal" Version="3.0.467" />
     <PackageReference Include="Microsoft.ServiceFabric.Services" Version="3.0.467" />
     <PackageReference Include="Microsoft.ServiceFabric.Services.Remoting" Version="3.0.467" />
-    <PackageReference Include="NUnit" Version="3.7.1" />
-    <PackageReference Include="NUnitLite" Version="3.7.1" />
+    <PackageReference Include="NUnit" Version="3.11.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.11.2" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
```
Comment (danielmarbach) [inline src/NServiceBus.Persistence.TestRunner/NServiceBus.Persistence.TestRunner.csproj]: Same remark as in https://github.com/Particular/NServiceBus.Persistence.ServiceFabric/pull/82#pullrequestreview-187663009

**Comment by danielmarbach** [comment]
Furthermore the build is failing because the test listener interface has changed in NUnit

**Pull request merged**